### ISSUE TITLE
[COGNITION]: GIBCT® Colmery 107 - Table caption mentions links. Should the table be updated for these instructions?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1646]
## Description

The school locations table doesn't announce itself properly to Safari + VoiceOver, and has some invalid HTML. It would be good to refactor this table for assistive device users. Screenshot and code sample below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Related Items
* https://app.zenhub.com/workspace/o/department-of-veterans-affairs/va.gov-team/issues/1640

## Acceptance Criteria

* As a user, I want the table instructions to match the table, and links be provided if necessary.

## Environment

* MacOS Mojave
* Safari
* VoiceOver
* https://staging.va.gov/gi-bill-comparison-tool/profile/11806124

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2019-09-09 at 1.03.27 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/214e20cf-4c0f-464b-900c-f01f1c73e80f)

**Comment by desireeturner** [comment]
@department-of-veterans-affairs/bah_uxteam Can you take a look at this issue and provide guidance regarding the question included in the screenshot around whether or not links should be included here?

**Comment by shawkey-daniel** [comment]
This issue was discussed prior to this defect. It was decided to remove the offending sentence, if there were no actual links provided. Since the story never was created, it should just be handled as part of this defect.

**Comment by NateJohnson-BAH** [comment]
Added conditional logic to only render the offending sentence when branches exist under a given institution. Otherwise, sentence is hidden and makes logical sense. 

PR link: https://github.com/department-of-veterans-affairs/vets-website/pull/10712

**Comment by delli-gatti-michael** [comment]
The reference to links has been conditionally removed if no links are present. This bug passes QA

#### With Links
![Screen Shot 2019-09-13 at 3.01.50 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/7b4200a1-b118-44a2-a5e3-637531787ebf)

#### Without Links
![Screen Shot 2019-09-13 at 3.06.41 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/5d53460f-ea5d-4a4c-a620-ca67bce7d0f8)

**Comment by 1Copenut** [comment]
@delli-gatti-michael This looks good to me too. I'll go ahead and close the ticket since this is a cognition issue and the code was easy to review in PR.
